### PR TITLE
Added jetrails/drone-cloudflare-caching@1.0.0 & jetrails/drone-cloudf…

### PR DESCRIPTION
**Pull request by null93**
Added two drone plugins to index:

- [jetrails/drone-cloudflare-caching](https://github.com/jetrails/drone-cloudflare-caching)
- [jetrails/drone-cloudflare-dns](https://github.com/jetrails/drone-cloudflare-dns)

### CODE REVIEW

File: content/JetRails/drone-cloudflare-caching/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+version: '1.0.0'
```
Comment (tboerger) [inline content/JetRails/drone-cloudflare-caching/index.md]: Drop that attribute please

File: content/JetRails/drone-cloudflare-dns/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+version: '1.0.0'
```
Comment (tboerger) [inline content/JetRails/drone-cloudflare-dns/index.md]: Please drop that attribute

**Comment by null93** [comment]
Made requested changes. Used `drone-plugins/drone-ansible` as a template.

**Comment by tboerger** [comment]
@xoxys maybe you can also review that

**Pull request merged**